cookbook/templates/default/service.yml.erb

```diff
@@ -0,0 +1,14 @@
+server:
+  type: simple
+  applicationContextPath: /baragon/v2
```

Comment:
another good candidate for a node setting (also app context path for agent as well)